cms/static/sass/views/_group-configuration.scss

```diff
@@ -19,206 +19,200 @@
       text-align: center;
     }
 
-    .no-group-configurations-content {
-      @extend %ui-well;
-      padding: ($baseline*2);
-      background-color: $gray-l4;
-      text-align: center;
-      color: $gray;
+    .no-content {
+      @extend %no-content;
     }
 
-    .new-button {
-      @extend %t-action3;
-      margin-left: $baseline;
+    .wrapper-groups {
+      margin-bottom: ($baseline*1.5);
 
-      .icon {
-        margin-right: ($baseline/2);
+      .title {
+        @extend %t-title4;
+        @extend %t-strong;
+        margin-bottom: ($baseline/2);
+      }
+
+      .copy {
+        @extend %t-copy-sub1;
       }
     }
 
-    .group-configuration {
+    .wrapper-collection {
       @extend %ui-window;
       position: relative;
       outline: none;
 
-      .group-configuration-details {
-        .wrapper-group-configuration {
-          padding: $baseline ($baseline*1.5);
+      &:hover .collection .actions {
+        opacity: 1.0;
+      }
 
-          .group-configuration-header {
-            margin-bottom: 0;
-            border-bottom: 0;
-          }
+      .collection-details {
+        padding: $baseline ($baseline*1.5);
+      }
 
-          .group-configuration-title {
-            @extend %t-title4;
-            @extend %t-strong;
-            overflow: hidden;
-            text-overflow: ellipsis;
-            margin-right: ($baseline*14);
+      .collection-header {
+        margin-bottom: 0;
+        border-bottom: 0;
+        padding-bottom: 0;
 
-            .group-toggle {
-              display: inline-block;
-              padding-left: $baseline;
-              color: $black;
+        .title {
+          @extend %t-title5;
+          @extend %t-strong;
+          overflow: hidden;
+          text-overflow: ellipsis;
```

Comment:
NIT: Can we use ``%cont-truncated`` (https://github.com/edx/edx-platform/blob/master/common%2Fstatic%2Fsass%2F_mixins.scss#L424) here?